README.md

```diff
@@ -33,7 +38,7 @@ of the familiar faces contributing to our projects:
 
 [![dwyl-community](https://user-images.githubusercontent.com/194400/28011265-a95f52d4-6559-11e7-823e-6133d947921a.jpg)](https://github.com/dwyl/who)
 
-The core team operate commercially through http://www.dwyl.io/ - an IT Services Company based in the United Kingdom. The commercial activity funds us personally and shapes the structure of our efforts including contribution prioritisation as well as our primary [technology stack](https://github.com/dwyl/technology-stack). 
+The core team operate commercially through http://www.dwyl.io/ - an IT Services Company based in the United Kingdom. The commercial activity funds us personally and shapes the structure of our efforts including contribution prioritisation as well as our primary [technology stack](https://github.com/dwyl/technology-stack).
```

Comment:
I'm in favour of _completely_ removing this sentence. It adds no value.
We need to focus on building the App and _completely_ phase out "IT Services".

Reply:
Definitely. As I mentioned, this whole README needs an overhaul 👍

Reply:
As much as this was clarifying in 2017, it is no longer relevant now. 👍 
Thanks for removing.  🔪